Gemfile

```diff
@@ -0,0 +1,20 @@
+source 'https://rubygems.org'
+
+gem 'rails', '4.2.5'
+gem 'sqlite3'
+gem 'sass-rails', '~> 5.0'
+gem 'uglifier', '>= 1.3.0'
+gem 'coffee-rails', '~> 4.1.0'
+gem 'jquery-rails'
+gem 'jbuilder', '~> 2.0'
+gem 'sdoc', '~> 0.4.0', group: :doc
+
+group :development, :test do
+  gem 'rspec-rails'
+  gem 'byebug'
```

Comment:
this should be replaced with `pry`. Our users haven't seen byebug I don't think.